src/modes/map/map_treasure.cpp

```diff
@@ -318,9 +318,19 @@ void TreasureSupervisor::_UpdateList()
             // If true, the drunes have been selected
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
-            _detail_textbox.SetDisplayText(VTranslate("With the additional %u drunes found in this treasure added, "
-                                                      "the party now holds a total of %u drunes.",
-                                                      _treasure->_drunes, GlobalManager->GetDrunes()));
+
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
Indentation is wrong, btw.
Also, I wouldn' put the sentence content in comment, but rather tell what it does if needed.